app/views/trains/edit.html.slim

```diff
@@ -1,3 +1,3 @@
 h1 Editing Train
-= render partial: 'form', locals: { url: train_path, method: :patch }
+= render partial: 'form', locals: { checked: train["active"], submit_text: "Update", url: train_path, method: :patch }
```

Comment:
i dont think you need to be doing `checked:` here. its too abstracted from where its actually being used.
put the logic in the form instead imo

Reply:
> i dont think you need to be doing `checked:` here. its too abstracted from where its actually being used.
> put the logic in the form instead imo

so it turns out, there will be the same radio button issue when an update form is submitted with errors

Reply:
regardless of if theres an issue or not, it shouldnt be done here imo, its too abstracted for no gain other then it being called `checked`
we need to look at ensuring the inputs are submitting as expected imo